config/bootstrap.php

```diff
@@ -33,11 +33,21 @@
  */
 require CORE_PATH . 'config' . DS . 'bootstrap.php';
 
+// You can remove this if you are confident that your PHP version is sufficient.
+if (version_compare(PHP_VERSION, '5.5.9') < 0) {
```

Comment:
would be nice if we could keep the version DRY, to only have a single source of change if we ever bump it.
Reading from composer file if existent would be cool. Maybe there are better options.